nrf_security/Kconfig

```diff
@@ -1322,6 +1322,11 @@ endif # MBEDTLS_RSA_C
 
 menu "SHA   - Secure Hash Algorithm"
 
+config MBEDTLS_MD5_C
+	bool
+	help
+	  MD5 hash functionality.
```

Comment:
Can you add the following statement in the help-text (after a newline)
"MD5 is not recommended for general use as a secure hash algorithm"